Nimble_Snapshots/HaveValidSnapshot.swift

```diff
@@ -66,7 +67,8 @@ public class FBSnapshotTest: NSObject {
         snapshotController.recordMode = record
         snapshotController.referenceImagesDirectory = referenceDirectory
         snapshotController.imageDiffDirectory = defaultImageDiffDirectory
-        snapshotController.usesDrawViewHierarchyInRect = usesDrawRect
+        // if an userInterfaceStyle is sent, we need to force usesDrawViewHierarchyInRect
+        snapshotController.usesDrawViewHierarchyInRect = userInterfaceStyle == nil ? usesDrawRect : true
```

Comment:
I would like to hear what more experienced in the library thinks about it.
I'm not sure about the side effects of using `usesDrawViewHierarchyInRect` or not.

Reply:
Could you please elaborate on this solution, I'm trying to understand why `usesDrawViewHierarchyInRect` should be always true when an `userInterfaceStyle` is set